src/lib/conditions.ts

```diff
@@ -0,0 +1,49 @@
+type TConditionCallback = () => boolean;
+
+export class Conditions {
```

Comment:
export it as `C`, should be shorthand and easy to use this. Also, why isn't this entire thing a namespace?